src/views/CardStack.js

```diff
@@ -571,6 +523,8 @@ class CardStack extends Component<DefaultProps, Props, void> {
     const SceneComponent = this.props.router.getComponentForRouteName(
       props.scene.route.routeName
     );
+    const extraProps = this.props.createExtraSceneProps &&
```

Comment:
I think we should avoid calling this callback from the render path. Instead I think it belongs in componentDidUpdate or componentWillUpdate. That way, we can clear old scenes that are gone